portal-web/test/functional/com/liferay/portalweb/macros/Navigator.macro

```diff
@@ -269,4 +269,8 @@
 	<command name="selectSpecificWindow">
 		<execute function="SelectWindow" locator1="title=${windowTitle}" />
 	</command>
+
```

Comment:
This command should be in a different macro file. If it is specific to Blogs, then it should be in a Blogs macro. If other components use it, maybe try PortletEntry.macro.